MAINTAINERS.yml

```diff
@@ -1031,6 +1031,21 @@ Documentation:
     labels:
         - "area: Wifi"
 
+"Drivers: Xen":
```

Comment:
but is this only concerning `drivers`? Because I think this is much more than only drivers.

Reply:
`Xen Platform` would make more sense in my opinion.

Reply:
You are right, any suggestions for this?

Reply:
@stephanosio good idea (ignore the previous one I sent), I'll change it

Reply:
Or let's wait for other comments, I'll fix them together:)